gazebo/rendering/Distortion.cc

```diff
@@ -91,6 +94,14 @@ namespace gazebo
       virtual void notifyMaterialRender(Ogre::uint32 _passId,
                                         Ogre::MaterialPtr& _material)
       {
+        // If more compositors are added to the camera in addition to the
+        // distortion compositor, an Ogre bug will cause the material for the
+        // last camera initialized to be used for all cameras. This workaround
+        // doesn't correct the material used, but it applies the correct texture
+        // to this active material.
+        _material->getTechnique(0)->getPass(_passId)->getTextureUnitState(1)->
+          setTexture(distortionTexture);
+      
```

Comment:
nit: I believe there is trailing whitespace on this line

Reply:
fixed